core/chains/evm/config/toml/defaults/zkSync_Sepolia.toml

```diff
@@ -1,18 +1,32 @@
 ChainID = '300'
 ChainType = 'zksync'
-FinalityDepth = 10
+# 200block ~ 20min concurrent with the l1_committed tag
```

Comment:
These are recent config changes added by @stackman27 4 months ago.
(same for mainnet)